modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/imagestack/v0.1.0.json

```diff
@@ -7,7 +7,7 @@
   ],
   "@type": "nxv:Schema",
   "import": [
-    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0",
```

Comment:
This is already fixed in the version v0.2.0. Also a deprecated schema should not be updated anymore.